contracts/test/failing-path.js

```diff
@@ -476,6 +476,44 @@ describe("DAO - Failing Paths", function () {
 				ctx.executeTx(executeProposalTx);
 			}, RUNTIME_ERR1009);
 		});
+
+		it("Should reject if fee paid by daoFundLsig for type == 1 (ALGO TRANSFER)", function () {
```

Comment:
I don't understand the tests, should it just be to execute the algo transfer and asa transfer .?